src/ports/postgres/modules/deep_learning/test/madlib_keras_iris.setup.sql_in

```diff
@@ -293,3 +293,36 @@ SELECT train_test_split('iris_data',     -- Source table
+$$::json,  NULL,
+ 'Sophie',
+                               'A simple model'
```

Comment:
This reminds me of something I'd been thinking for a while--it seems like in these dev-check tests we should be starting from python code rather than json, or at least check in the python code somewhere that generated the json.

Otherwise, every time the keras or tensorflow version increases, we'll have to re-generate the json from python code, and if it's not checked in then we may not even be able to find the code, and have to start from scratch again.

Probably a separate story though.